### PR TITLE
Bugzilla doc text for 4.3.22 release

### PR DESCRIPTION
**Pull request by jeana-redhat**

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@codyhoag would love your review when you have a chance!

File: release_notes/ocp-4-3-release-notes.adoc [hunk 1]
```diff
@@ -2366,3 +2366,95 @@ After this one-time manual restart, subsequent upgrades and rotations will
...
+release:
+
+link:https://access.redhat.com/solutions/5099891[{product-title} 4.3.22 container image list]
+
+[[ocp-4-3-22-bug-fixes]]
+==== Bug Fixes
+
+* Previously, installations on the s390x and ppc64le architectures would not
+complete successfully because the Samples Operator was failing to report its
+version when running on those architectures.  This problem has been fixed, and
```
Comment (codyhoag) [inline release_notes/ocp-4-3-release-notes.adoc:2392]: There's an extra space before `This` to begin the sentence.

File: release_notes/ocp-4-3-release-notes.adoc [hunk 2]
```diff
@@ -2366,3 +2366,95 @@ After this one-time manual restart, subsequent upgrades and rotations will
...
+these installations now complete successfully.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1779934[*BZ#1779934*])
+
+* With this update, several environment variables involving TLS certificates and
+ the ETCDCTL API version are written into /root/.profile. As a result, when
```
Comment (codyhoag) [inline release_notes/ocp-4-3-release-notes.adoc:2397]: I'd surround `/root/.profile` with backticks.

File: release_notes/ocp-4-3-release-notes.adoc [hunk 3]
```diff
@@ -2366,3 +2366,95 @@ After this one-time manual restart, subsequent upgrades and rotations will
...
+ users perform oc rsh, the etcdctl commands now work without having to set these
```
Comment (codyhoag) [inline release_notes/ocp-4-3-release-notes.adoc:2398]: Backticks around `oc rsh` as well.

File: release_notes/ocp-4-3-release-notes.adoc [hunk 4]
```diff
@@ -2366,3 +2366,95 @@ After this one-time manual restart, subsequent upgrades and rotations will
...
+ variables manually.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1801430[*BZ#1801430*])
+
+* To prevent problems with one of the registry pods from affecting registry
```
Comment (codyhoag) [inline release_notes/ocp-4-3-release-notes.adoc:2402]: s/pods/Pods

File: release_notes/ocp-4-3-release-notes.adoc [hunk 5]
```diff
@@ -2366,3 +2366,95 @@ After this one-time manual restart, subsequent upgrades and rotations will
...
+messages when appropriate. As a result, users get better guidance after
+successive failed attempts on what is needed to fix problems with
+ImageStreamImport.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1833019[*BZ#1833019*])
+
+* Previously, the cluster-network-operator did not remove deprecated security
+group rules on Kuryr bootstrapping when they were replaced by new rules. As a
+result, the deprecated rules were left in place during OCP upgrades between
+4.3.z releases. The cluster-network-operator is now updated to remove deprecated
+security group rules so that pods continue to have the correct host VM access
```
Comment (codyhoag) [inline release_notes/ocp-4-3-release-notes.adoc:2437]: s/pods/Pods

**Review by codyhoag — COMMENTED** [review]
Just a couple tiny wordsmithing suggestions. Everything else looks great! :+1:

File: release_notes/ocp-4-3-release-notes.adoc (hunk 2 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-3-release-notes.adoc:2397]: Oops - I intended to but forgot to format after pasting :) Good catch!

**Pull request merged**